internal/statemachine/helper.go

```diff
@@ -594,9 +594,13 @@ func generateDebootstrapCmd(imageDefinition imagedefinition.ImageDefinition, tar
 	debootstrapCmd := execCommand("debootstrap",
 		"--arch", imageDefinition.Architecture,
 		"--variant=minbase",
-		"--include=ca-certificates", // ca-certificates is needed to use PPAs
 	)
 
+	if imageDefinition.Customization != nil && len(imageDefinition.Customization.ExtraPPAs) > 0 {
+		// ca-certificates is needed to use PPAs
```

Comment:
The pedant in me wants to argue with this. I think it's only _required_ when using private ppas, because ppa content is available over plain http too. But well.